.github/workflows/swagger.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+      - DTSRD-3562_Swagger_Publishing
```

Comment:
possibly remove branch to match other PRs

Reply:
removed

Reply:
removed